sql/service/src/main/java/org/apache/spark/sql/service/cli/Type.java

```diff
@@ -22,14 +22,14 @@
 import java.util.Locale;
 
 import org.apache.spark.sql.service.rpc.thrift.TTypeId;
-import org.apache.spark.sql.types.DecimalType;
+import org.apache.spark.sql.types.*;
 
 /**
  * Type.
  *
  */
 public enum Type {
-  NULL_TYPE("VOID",
+  NULL_TYPE(NullType$.MODULE$.sql(),
```

Comment:
I think this is ok and doesn't break protocol compatibility, but I wouldn't be sure without a test for that.